lib/acts-as-messageable/scopes.rb

```diff
@@ -12,8 +12,9 @@ module ClassMethods
       def initialize_scopes
         scope :are_from,          lambda { |*args| where(:sent_messageable_id => args.first, :sent_messageable_type => args.first.class.name) }
         scope :are_to,            lambda { |*args| where(:received_messageable_id => args.first, :received_messageable_type => args.first.class.name) }
-        scope :search,            lambda { |*args|  where("body like :search_txt or topic like :search_txt",:search_txt => "%#{args.first}%")}
-        scope :connected_with,    lambda { |*args|  where("(sent_messageable_type = :sent_type and
+        scope ActsAsMessageable.search_scope_name,
+                                  lambda { |*args| where("body like :search_txt or topic like :search_txt", :search_txt => "%#{args.first}%") }
+        scope :connected_with,    lambda { |*args| where("(sent_messageable_type = :sent_type and
```

Comment:
Line is too long. [97/80]<br>Block body expression is on the same line as the block start.